subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -37,8 +37,9 @@ static struct mqtt_client *c;
 static enum execution_status execution_state = AWS_JOBS_QUEUED;
 static enum fota_status fota_state = NONE;
 
-/* Document version starts at 1 and is incremented with each accepted update */
-static u32_t doc_version_number = 1;
+/* Document version starts is read out from the job execution document  and is
+ * incremented with each accepted update */
+static u32_t doc_version_number;
```

Comment:
Should we change this to something which better maps it to the field it represents?

AFAICS this is the `versionNumber` field in the `jobExecution` data type.

Reply:
Change it to execution_version_number will add data structures in another commit